### ISSUE TITLE
data-inview: How to reverse?

### ISSUE THREAD

**Issue by chyne** [wet-boew/wet-boew#7915]
I have a section that is fading out when another section is fully out of view. How do I make it fade in when a section is fully out of view (opposite of current bahaviour).

The documentation says to add "reverse" class, but that doesn't seem to be working for me.

**Comment by duboisp** [comment]
The ```reverse``` option only reverse the transition, not the behavior of when the transition should occur.

Try to add the option ```show-none```, I think that is want you are looking for. It is documented under the heading "Configuration options"

**Comment by chyne** [comment]
I think reverse is what I am looking for. The timing is already correct (I already have show-none so when my element, before, is fully out of view). What I want is it to fade in, instead of fading out.

**Comment by chyne** [comment]
I.e. take the "section that fades the next section" example in the docs and have it fade in instead of out when the first section go's out of view.

**Comment by kodecount** [comment]
It is using CSS for the animations. You could override these classes if you want a different animation.

**Comment by chyne** [comment]
Based on my reading of the data-inview code, there is no support for "reverse".

**Comment by duboisp** [comment]
@chyne Have you tried to move the element that call the ```wb-inview``` with the ```data-inview``` attribute before or after the section your try to fade in/out. You should see different behavior.

**Comment by chyne** [comment]
The element I have wb-inview/data-inview on is one of the header sections, so I can't practically move my fading section before it.

The code for the inview plugin is hard coded to either **open** overlays, or fade **out** elements when the element goes out of view. You can't **close** overlays or fade **in** elements, despite what the documentation says.

Inserting the following before line 125 would, I believe allow it to fade in:

`if ($dataInView.hasClass("reverse")) {
    show = !show;
}`

In the meantime, I'll probably have to hook into the events raised to do my own transitions.

TL;DR: The documentation is wrong, there is no support for "reverse" transitions.

**Comment by duboisp** [comment]
@chyne I looked closer and I found that only ```pop```, ```slide```, ```slidefade``` and ```slidevert``` that support the reverse transition. 

I will submit a PR that would fix your issue and add the reverse fade in/out
